css/style.css

```diff
@@ -220,14 +225,18 @@ main {
   border-radius: 2px;
 }
 
-.toggle-inner {
+input[type="checkbox"] {
+  display: none;
```

Comment:
Display none will remove all the accessibility benefits of this, use `opacity: 0;`